common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -1,35 +1,31 @@
 @()(implicit request: RequestHeader)
 
 @import common.{Edition, LinkTo}
+@import views.support.RenderClasses
 
-<div class="edition-picker js-edition-picker-container" data-component="edition-picker">
+<li class="navigation-group__item">
+    <details>
+        <summary class="main-navigation__item__button"
+        data-link-name="nav2 : edition picker">
 
-    <input class="edition-picker__button js-enhance-checkbox"
-        type="checkbox" id="edition-picker"
-        aria-controls="edition-picker__dropdown">
+            <i class="main-menu__icon main-menu__icon--chevron personalisation__icon"></i>
```

Comment:
It is actually done with css, the styling was added a while ago, this is just using the class name